### PR TITLE
Pipeline release fixes

### PR DESCRIPTION
**Pull request by andrewballantyne**
Closes: #1222

## Description

Misc fixes to help smooth out the first release.

Major ones are:
 - Support a better Job => Run on the Triggered Runs table by using Job details in rendering of the rows
 - Support a "what if the Data Connection is bad" error state (5mins before we drop into it -- should get better next sprint)

### Screenshots

image
image (3)

## Test Impact

None. https://github.com/opendatahub-io/odh-dashboard/issues/1184

## Request review criteria:

- [x] The commits have meaningful messages (squashes happen on merge by the bot).
- [ ] Included any necessary screenshots or gifs if it was a UI change.
- [ ] Included tags to the UX team if it was a UI/UX change.
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work
- [x] The developer has added tests or explained why testing cannot be added (unit tests & storybook for related changes)

### CODE REVIEW

File: frontend/src/api/pipelines/custom.ts [hunk 1]
```diff
@@ -68,7 +68,10 @@ export const listPipelines: ListPipelinesAPI = (hostPath) => (opts, count) =>
   );
 
 export const listPipelineRuns: ListPipelinesRunAPI = (hostPath) => (opts) =>
-  handlePipelineFailures(proxyGET(hostPath, '/apis/v1beta1/runs', {}, opts));
+  handlePipelineFailures(
+    // eslint-disable-next-line camelcase
+    proxyGET(hostPath, '/apis/v1beta1/runs', { sort_by: 'created_at desc' }, opts),
```
Comment (andrewballantyne) [inline frontend/src/api/pipelines/custom.ts:73]: Log a follow up issue for this.

File: frontend/src/concepts/pipelines/context/PipelinesContext.tsx [hunk 2]
```diff
@@ -191,3 +207,34 @@ export const ViewServerModal = ({ isOpen, onClose }: { isOpen: boolean; onClose:
     />
   );
 };
+
+export const PipelineServerTimedOut: React.FC = () => {
```
Comment (andrewballantyne) [inline frontend/src/concepts/pipelines/context/PipelinesContext.tsx:211]: Need to handle this better -- waiting on more info from backend -- log issue.

File: backend/src/utils/prometheusUtils.ts [hunk 3]
```diff
@@ -31,7 +31,7 @@ const callPrometheus = async <T>(
   const url = `${host}/api/v1/${queryType}?${query}`;
 
   fastify.log.info(`Prometheus query: ${query}`);
-  return proxyCall(fastify, request, { method: 'GET', url })
+  return proxyCall(fastify, request, { method: 'GET', url, rejectUnauthorized: false })
```
Comment (andrewballantyne) [inline backend/src/utils/prometheusUtils.ts:34]: Part of the Pipeline proxy stuff was to drop this from the original implementation -- I think it is causing issues with QE clusters -- adding it back.

File: frontend/src/concepts/pipelines/content/ViewPipelineServerModal.tsx [hunk 4]
```diff
@@ -9,7 +9,6 @@ import {
   Title,
 } from '@patternfly/react-core';
 import { usePipelinesAPI } from '~/concepts/pipelines/context';
-import EnsureAPIAvailability from '~/concepts/pipelines/EnsureAPIAvailability';
```
Comment (andrewballantyne) [inline frontend/src/concepts/pipelines/content/ViewPipelineServerModal.tsx:12]: Just dropped this -- while the server is loading, you may want to look at your creds you used.

**Review by Gkrumbach07 — APPROVED** [review]
lgtm

File: frontend/src/api/pipelines/custom.ts (hunk 1 above)
Comment (andrewballantyne) [reply frontend/src/api/pipelines/custom.ts:73]: https://github.com/opendatahub-io/odh-dashboard/issues/1223

File: frontend/src/concepts/pipelines/context/PipelinesContext.tsx (hunk 2 above)
Comment (andrewballantyne) [reply frontend/src/concepts/pipelines/context/PipelinesContext.tsx:211]: https://github.com/opendatahub-io/odh-dashboard/issues/1224

**Pull request merged**